__tests__/components/organisms/ConversationWindow.spec.js

```diff
@@ -0,0 +1,60 @@
+import { shallowMount } from "@vue/test-utils";
+import ConversationWindow from "../../../src/components/organisms/ConversationWindow";
+import TextInput from "../../../src/components/atoms/TextInput";
+import MessageHeader from "../../../src/components/atoms/MessageHeader";
+import { createStore, useStore } from "vuex";
+import { i18n } from "../../../i18n";
+import chatMessages from "../../../src/store/modules/chatMessages";
+jest.mock("../../../src/store/modules/chatMessages");
+import emails from "../../../src/store/modules/emails";
+jest.mock("../../../src/store/modules/emails");
+import inbox from "../../../src/store/modules/inbox";
+jest.mock("../../../src/store/modules/inbox");
+
+describe("ConversationWindow component", () => {
```

Comment:
Could we check to make sure that the `ConversationMessage` component is rendered as well? Just like we did with line 37 and 41.

Reply:
Added check for ConversationMessages. Quite beneficial as it checks if getter returns value that v-for can render ConversationMessages